spikingjelly/activation_based/functional/net_config.py

```
@@ -1,11 +1,34 @@
 
 from .. import base
 
-__all__ = ["reset_net", "set_step_mode", "set_backend", "detach_net"]
+__all__ = [
+    "collect_reset_modules",
+    "reset_collected_modules",
+    "reset_net",
+    "set_step_mode",
+    "set_backend",
+    "detach_net",
+]
```

Ruff (0.15.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)